### ISSUE TITLE
test_key_add_userid and test_ffi_security_profile failure

### ISSUE THREAD

**Issue by andreasstieger** [rnpgp/rnp#2182]
`test_key_add_userid` and test_ffi_security_profile fail in similar ways 0.17.0.

This is probably related to an update of a dependency in the distribution, we need to yet determine which one.

current log: https://build.opensuse.org/package/live_build_log/security:privacy/rnp/openSUSE_Tumbleweed/x86_64
current sources https://build.opensuse.org/package/show/openSUSE:Factory/rnp

````
[  127s]         Start  76: rnp_tests.test_key_add_userid
[  127s]  74/260 Test  #76: rnp_tests.test_key_add_userid .................................................***Failed    0.11 sec
[  127s] Note: Google Test filter = rnp_tests.test_key_add_userid
[  127s] [==========] Running 1 test from 1 test suite.
[  127s] [----------] Global test environment set-up.
[  127s] [----------] 1 test from rnp_tests
[  127s] [ RUN      ] rnp_tests.test_key_add_userid
[  127s] /home/abuild/rpmbuild/BUILD/rnp-v0.17.0/src/tests/key-add-userid.cpp:75: Failure
[  127s] Expected equality of these values:
[  127s]   (base_expiry)
[  127s]     Which is: 1234567890
[  127s]   (key->expiration())
[  127s]     Which is: 0
[  127s] [  FAILED  ] rnp_tests.test_key_add_userid (94 ms)
[  127s] [----------] 1 test from rnp_tests (94 ms total)
[  127s] 
[  127s] [----------] Global test environment tear-down
[  127s] [==========] 1 test from 1 test suite ran. (94 ms total)
[  127s] [  PASSED  ] 0 tests.
[  127s] [  FAILED  ] 1 test, listed below:
[  127s] [  FAILED  ] rnp_tests.test_key_add_userid
[  127s] 
[  127s]  1 FAILED TEST
[  127s] 
````

````
[  150s] 212/260 Test #212: rnp_tests.test_ffi_security_profile ...........................................***Failed    0.52 sec
[  150s] Note: Google Test filter = rnp_tests.test_ffi_security_profile
[  150s] [==========] Running 1 test from 1 test suite.
[  150s] [----------] Global test environment set-up.
[  150s] [----------] 1 test from rnp_tests
[  150s] [ RUN      ] rnp_tests.test_ffi_security_profile
[  150s] /home/abuild/rpmbuild/BUILD/rnp-v0.17.0/src/tests/ffi.cpp:5956: Failure
[  150s] Expected equality of these values:
[  150s]   (from)
[  150s]     Which is: 1705629600
[  150s]   (1547856000)
[  150s]     Which is: 1547856000
[  150s] [  FAILED  ] rnp_tests.test_ffi_security_profile (507 ms)
[  150s] [----------] 1 test from rnp_tests (507 ms total)
[  150s] 
[  150s] [----------] Global test environment tear-down
[  150s] [==========] 1 test from 1 test suite ran. (507 ms total)
[  150s] [  PASSED  ] 0 tests.
[  150s] [  FAILED  ] 1 test, listed below:
[  150s] [  FAILED  ] rnp_tests.test_ffi_security_profile
[  150s] 
[  150s]  1 FAILED TEST
[  150s] 
````

**Comment by ni4** [comment]
@andreasstieger Yeah, it's because of SHA1 deprecation for key signatures, and going to be fixed via PR #2180

**Comment by andreasstieger** [comment]
Confirming that #2180 fixes this.
